### ISSUE TITLE
Inline Jinja template not evaluated correctly for includes

### ISSUE THREAD

**Issue by aw1cks** [ansible-community/ansible-lint#2011]
##### Summary
When using an inline Jinja template to include another file, the linter throws an error.

This doesn't seem to happen when using the `{{ <expression> }}` syntax, only when using something like this:

```jinja
{%- if condition -%}
value1
{%- else -%}
value2
{%- endif -%}
```

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version

~ » ansible --version
ansible [core 2.12.2]
  config file = None
  configured module search path = ['/Users/alexw/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/homebrew/Cellar/ansible/5.3.0/libexec/lib/python3.10/site-packages/ansible
  ansible collection location = /Users/alexw/.ansible/collections:/usr/share/ansible/collections
  executable location = /opt/homebrew/bin/ansible
  python version = 3.10.2 (main, Feb  2 2022, 05:51:25) [Clang 13.0.0 (clang-1300.0.29.3)]
  jinja version = 3.0.3
  libyaml = True

ansible-lint --version

~ » ansible-lint --version
WARNING: PATH altered to include /opt/homebrew/Cellar/ansible-lint/5.4.0/libexec/bin
ansible-lint 5.4.0 using ansible 2.12.2

```

- ansible installation method: homebrew
- ansible-lint installation method: homebrew

(N.B. same behaviour is observed in a fresh venv installing latest pip packages)

##### OS / ENVIRONMENT

Tested on MacOS and Ubuntu 18.04

##### STEPS TO REPRODUCE

This doesn't work:
```yaml
~/test » cat play.yaml
---
- name: Testing inline Jinja2
  hosts: localhost
  tasks:
    - name: Conditionally include our task
      include_tasks:
        file: >-
          {%- if ansible_os_family == 'Debian' -%}
          task_debian.yaml
          {%- else -%}
          task_other.yaml
          {%- endif -%}
~/test » ansible-lint play.yaml
WARNING: PATH altered to include /opt/homebrew/Cellar/ansible-lint/5.4.0/libexec/bin
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: play.yaml
WARNING  Listing 1 violation(s) that are fatal
load-failure: [Errno 2] No such file or directory: "/Users/alexw/test/{'file':" (filenotfounderror)
{'file'::1

You can skip specific rules or tags by adding them to your configuration file:
# .ansible-lint
warn_list:  # or 'skip_list' to silence them completely
  - load-failure  # Failed to load or parse file

Finished with 1 failure(s), 0 warning(s) on 2 files.
~/test »
```

Whereas this does:
```shell
~/test » cat play.yaml
---
- name: Testing inline Jinja2
  hosts: localhost
  tasks:
    - name: Conditionally include our task
      include_tasks:
        file: >-
          {{ 'task_debian.yaml' if ansible_os_family == 'Debian' else 'task_other.yaml' }}
~/test » ansible-lint play.yaml
WARNING: PATH altered to include /opt/homebrew/Cellar/ansible-lint/5.4.0/libexec/bin
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: play.yaml
~/test »
```

##### Desired Behavior

Both template syntaxes should be supported.

##### Actual Behavior

```shell
~/test » ansible-lint play.yaml
WARNING: PATH altered to include /opt/homebrew/Cellar/ansible-lint/5.4.0/libexec/bin
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: play.yaml
WARNING  Listing 1 violation(s) that are fatal
load-failure: [Errno 2] No such file or directory: "/Users/alexw/test/{'file':" (filenotfounderror)
{'file'::1

You can skip specific rules or tags by adding them to your configuration file:
# .ansible-lint
warn_list:  # or 'skip_list' to silence them completely
  - load-failure  # Failed to load or parse file

Finished with 1 failure(s), 0 warning(s) on 2 files.
~/test »
```
